src/core/server/path/index.ts

```diff
@@ -54,6 +58,13 @@ function findFile(paths: string[]) {
  * @internal
  */
 export const getConfigPath = () => findFile(CONFIG_PATHS);
+
+/**
+ * Get the path where the config directories are stored
+ * @internal
+ */
+export const getConfigDirectory = () => findFile(CONFIG_DIRECTORIES);
```

Comment:
Misleading comment. This resolves to the actual config directory.

Outside of the PR, but the `getConfigPath` description is also wrong (`Get the path where the config files are stored`), as it resolves the path to the config file itself.